.circleci/docker/common/install_user.sh

```diff
@@ -3,8 +3,9 @@
 set -ex
 
 # Mirror jenkins user in container
-echo "jenkins:x:1014:1014::/var/lib/jenkins:" >> /etc/passwd
-echo "jenkins:x:1014:" >> /etc/group
+# jenkins user as ec2-user should have the same user-id
+echo "jenkins:x:1000:1000::/var/lib/jenkins:" >> /etc/passwd
+echo "jenkins:x:1000:" >> /etc/group
```

Comment:
How did you know it was 1000? Just curious.

Reply:
1000 is a default first non-system user account on many Linux systems, RedHat even has a nice article about it: https://www.redhat.com/sysadmin/user-account-gid-uid